sympy/physics/hydrogen.py

```diff
@@ -87,15 +91,18 @@ def Psi_nlm(n, l, m, r, phi, theta, Z=1):
     Returns the Hydrogen wave function psi_{nlm}. It's the product of
     the radial wavefunction R_{nl} and the spherical harmonic Y_{l}^{m}.
 
-    n, l, m
+    Parameters
+    ==========
+
+    n, l, m :
```

Comment:
as mentioned above, I believe this should be on different lines.

Reply:
Changed.